views/settings/page-sections/dashboard.php

```diff
@@ -80,7 +80,14 @@
 						<span class="wpr-title3"><?php esc_html_e( 'License', 'rocket' ); ?></span>
 						<span class="wpr-infoAccount wpr-isValid" id="wpr-account-data">
 							<?php echo esc_html( $data['customer_data']['license_type'] ); ?>
-						</span><br>
+						</span>
+						<?php if ( ! $data['customer_data']['is_from_one_dot_com'] ) : ?>
```

Comment:
shouldn't it be the opposite, displaying the content if the value is true?

Reply:
Ahhhhh!
I must have slept off